Monika-After-Story/MonikaModDev
# Better db functions

This adds and refines the all-event database for much easier access and adding of new events.

# Key features
* All databases now have a 3-character string code associatd with it. 
* new functions:
    * `mas_lockEVL` / `mas_unlockEVL` / `mas_showEVL` / `mas_hideEVL`
    * these functions take event labels and the string code along with other appropraite params. (now you dont have to pass in eventdb. anymore)
* `addEvent` changed so it now accepts a code instead of eventdb. leads to easier to read `addEvent` lines.

I have updated all non-event events to use the code instead of eventdb.

# Testing
1. verify the new functions work.
2. verify nothing else is broken
